src/components/Tweet.css

```diff
@@ -10,3 +10,16 @@
 .Tweet__header {
   position: relative;
 }
+
+.Tweet_image {
+  border-radius: 5px;
+  /*width: 95%;*/
+  float: right;
```

Comment:
The tweet images are overflowing the available space for larger images. This could be avoided by putting them in a container div that has fixed dimensions with `overflow:hidden;` and you set the rounded borders on that container div. Even if you stick with just an img tag, you can specify a `max-width` to avoid the overflowing issue.